styles/background.scss

```diff
@@ -181,12 +224,24 @@ html body.dark_mode_on {
     .subcategory-link-bar--subcategory-in-dropdown--hdpz8:hover,
   .header--header--1SLKV .js-header-search-field::placeholder,
   .ud-btn-secondary:active,
-  .ud-btn-secondary:hover {
+  .ud-btn-secondary:hover,
+  .item-card-module--item-card-title--S729p:hover {
     color: var(--hover-font-color);
   }
-  .ud-footer .inverted-link {
+
+  .ud-footer .inverted-link,
+  .smart-bar-module--smart-bar__title--TnwIQ,
```

Comment:
The smartbar on top of the page changes colors a lot. Right now for me the bar is purple with a purple text color. Therefore, it is hard to read. As far as i have seen, it is sometimes purple and sometimes yellow, but i don't know if that is all colors.

Reply:
![image](https://github.com/Pachciar3/udemy-dark-mode-extension/assets/101325615/128e8a23-1426-461a-88a7-29b8bc5b1966)
As you can see it is currently yellow. It changes from time to time. Right now the textcolor is purple and with a purple background it is hard to read. If i run into it when it is purple i will send another image.

Reply:
![image](https://github.com/Pachciar3/udemy-dark-mode-extension/assets/43442271/db770b3f-6d37-410d-88d5-86ad75515acf)
I found the code with all variants. I tested, I think now it should be easier to read in all variants